### PR TITLE
Some new books for the project

### PR DESCRIPTION
**Pull request by switchhax**
To fill in some stuff. I added also some spaces like described in the rules. The german list is shorter, because i thought just add "chapters" for topics you can fill with stuff.
If somethings wrong, i´m going to rework that stuff.

### CODE REVIEW

File: free-science-books.md [hunk 1]
```diff
@@ -1,14 +1,31 @@
 # Subject areas (In order of LIBRARY OF CONGRESS CLASSIFICATION)
 
 ## Science (General)   (Q)
+* [Fundamentals of Piano Practice](http://www.pianopractice.org/)
```
Comment (eshellman) [inline free-science-books.md]: Nice book, but not sure I understand why this would be a "Science" book.

File: free-science-books.md [hunk 2]
```diff
@@ -1,14 +1,31 @@
...
+
 ## Mathematics (QA)
+
 ## Astronomy   (QB)
+
 ## Physics (QC)
+* [The Feynman Lectures on Physics](http://feynmanlectures.caltech.edu/)
+* [Quantum Diaries](http://www.quantumdiaries.org/)
+
```
Comment (eshellman) [inline free-science-books.md]: This is a blog, not a book.

File: free-science-books.md [hunk 3]
```diff
@@ -1,14 +1,31 @@
...
 
 ## Science (General)   (Q)
+* [Fundamentals of Piano Practice](http://www.pianopractice.org/)
+
 ## Mathematics (QA)
+
 ## Astronomy   (QB)
+
 ## Physics (QC)
+* [The Feynman Lectures on Physics](http://feynmanlectures.caltech.edu/)
```
Comment (eshellman) [inline free-science-books.md]: My fave!

File: free-science-books.md [hunk 4]
```diff
@@ -1,14 +1,31 @@
...
 ## Natural history (QH)
+
 ## Botany  (QK)
+
 ## Zoology (QL)
+
 ## Human anatomy   (QM)
+
 ## Physiology (QP)
+* [Biochemical Pathways](http://biochemical-pathways.com/#/map/1)
```
Comment (eshellman) [inline free-science-books.md]: This is poster, (I have a copy!) not a book.

File: free-science-books.md [hunk 5]
```diff
@@ -1,14 +1,31 @@
...
+* [Brain Basics](http://www.nimh.nih.gov/health/educational-resources/brain-basics/brain-basics.shtml)
```
Comment (eshellman) [inline free-science-books.md]: More of web page than a book. But NLM has a great list of free books here: http://www.ncbi.nlm.nih.gov/books/browse/

File: free-science-books-de.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+
+##Informatik
+* [Informatik-Schule](http://www.inf-schule.de/)
+* [Schule Regensburg - Informatik](http://www.schulen.regensburg.de/wvsgym/images/Faecher/Informatik/)
+
+##Mathe
+* [Mathe Prisma](http://www.matheprisma.uni-wuppertal.de/index.htm)
+
+##Musik
+* [Klavier spielen - Grundlagen, Übungen, Praxistipps](http://foppde.uteedgar-lins.de/foppde.html#copy)
```
Comment (eshellman) [inline free-science-books-de.md]: not really science?

File: free-science-books-de.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+
+## Elektronik
+* [Das Elektronik-Labor - Labortagebuch](http://www.elektronik-labor.de/Labortagebuch/Labortagebuch.html)
+
+##Informatik
+* [Informatik-Schule](http://www.inf-schule.de/)
+* [Schule Regensburg - Informatik](http://www.schulen.regensburg.de/wvsgym/images/Faecher/Informatik/)
+
+##Mathe
+* [Mathe Prisma](http://www.matheprisma.uni-wuppertal.de/index.htm)
```
Comment (eshellman) [inline free-science-books-de.md]: not very bookish

File: free-science-books-de.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
+# Fachrichtungen
+
+## Elektronik
+* [Das Elektronik-Labor - Labortagebuch](http://www.elektronik-labor.de/Labortagebuch/Labortagebuch.html)
```
Comment (eshellman) [inline free-science-books-de.md]: a laboratory notebook?

File: free-science-books-de.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+
+##Informatik
```
Comment (eshellman) [inline free-science-books-de.md]: don't these belong in free-programming-books instead?

File: free-science-books.md (hunk 1 above)
Comment (switchhax) [inline free-science-books.md]: Well Music can be considered a science depending on point of view.

File: free-science-books.md (hunk 2 above)
Comment (switchhax) [inline free-science-books.md]: But a great learning resource. I envisioned the whole more like an awesome list of learning resources concerning sciences. But ok only books.

File: free-science-books.md (hunk 3 above)
Comment (switchhax) [inline free-science-books.md]: Nice! I don´t understand much about it cause i´m still going to highschool

File: free-science-books.md (hunk 4 above)
Comment (switchhax) [inline free-science-books.md]: Also a great learning resource.

File: free-science-books.md (hunk 5 above)
Comment (switchhax) [inline free-science-books.md]: Well in the free-programming-book there are many books in the form of a webpage.

File: free-science-books-de.md (hunk 6 above)
Comment (switchhax) [inline free-science-books-de.md]: Again. Dpending on the point of view you can consider Music, Electronics and Mathematics a science or not.

File: free-science-books-de.md (hunk 7 above)
Comment (switchhax) [inline free-science-books-de.md]: They have some kind of chapters and sub-chapters. So yes. Not quite a book.

File: free-science-books-de.md (hunk 8 above)
Comment (switchhax) [inline free-science-books-de.md]: Yes, why not?

File: free-science-books-de.md (hunk 9 above)
Comment (switchhax) [inline free-science-books-de.md]: Yes, the "Informatik" respectively Computer Sciences part.

**Comment by eshellman** [comment]
This is a very useful pull request, because it highlights a bunch of the scope issues we'll face. We'll see if anyone else has opinions, but I suggest:

1. If a book would be catalogued in an LC category other than Science, Medicine, or Technology, for example, "MUSIC AND BOOKS ON MUSIC" http://www.loc.gov/aba/cataloging/classification/lcco/lcco_m.pdf
then It's out of scope for this list.
2. If a non-book resource is proposed, for example "screencasts", "interactive tutorials", etc., following the example of free-programming-books, we should encourage these resources to be included in a separate page dedicate to that sort of resource.
3. If a book or resource is a "free-programming-book" or resource, it should go there. Math books might go both places, but not computer science.
4. @switchhax , you and Feynmann deserve to be the inaugural book. resubmit your PR with just The Lectures, and you'll have the very first book submission!

**Comment by switchhax** [comment]
I think you should point out these 3 "rules"  on the CONTRIBUTING.md.

**Comment by eshellman** [comment]
added scope guidelines to CONTRIBUTING.md. Thanks again.

**Pull request closed without merging**